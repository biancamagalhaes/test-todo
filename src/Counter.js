import React, { Component } from "react";

function App(props) {
  return (
    <div className="App">
      <h1>VocÃª tem {props.count * 2} counters</h1>
    </div>
  );
}

export default App;
