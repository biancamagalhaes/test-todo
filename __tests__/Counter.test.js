import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Counter from "../src/Counter";

describe("Testing App Component", () => {
  it("should render correctly", () => {
    const wrapper = shallow(<Counter count={10} />);

    expect(toJson(wrapper)).toMatchSnapshot();
    wrapper.setProps({ count: 5 });
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
