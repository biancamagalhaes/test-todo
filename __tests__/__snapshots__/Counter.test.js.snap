// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing App Component should render correctly 1`] = `
<div
  className="App"
>
  <h1>
    Você tem 
    20
     counters
  </h1>
</div>
`;

exports[`Testing App Component should render correctly 2`] = `
<div
  className="App"
>
  <h1>
    Você tem 
    10
     counters
  </h1>
</div>
`;
